api/ls_ganap/main_events/urls.py

```diff
@@ -7,6 +7,7 @@
 urlpatterns = [
     path('events/', event_view.EventList.as_view()),
     path('events/<int:pk>/', event_view.EventDetail.as_view()),
```

Comment:
Good job on this one, it displays the list of event logistics, tied to the event